src/details-page.js

```diff
@@ -0,0 +1,165 @@
+const loader = document.getElementsByClassName('loader')[0];
+const detailsSlot = document.getElementById('details');
+
+// Extract CCA3 country code from current window's url params and remove loader
+const searchStr = location.search.substring(1);
```

Comment:
If you have a lot of code that need to be explained by adding a comment it is better to extract this code into a function and call the function